roles/postgres/molecule/default/verify.yml

```diff
@@ -0,0 +1,70 @@
+
+---
+- name: postgres Tests
+  hosts: all
+  become: yes
+  vars:
+    activemq_version: "5.18.7"
+    activemq_binary_path: "/opt/apache-activemq-{{ activemq_version }}/bin/activemq"
+    config_folder: "/etc/opt/alfresco"
+    activemq_password: "activemq"
+  tasks:
+  - name: Gather service facts
+    service_facts:
+  - name: Print ActiveMQ password
+    debug:
+      msg: "ActiveMQ password is: {{ inventory_hostname }}"  
+  - name: Ensure PostgreSQL service is running and enabled
+    fail:
+      msg: "PostgreSQL service is not running or enabled!"
+    when: 
+      - ansible_facts.services["postgresql-15.service"].state != "running"
+      - ansible_facts.services["postgresql-15.service"].status != "enabled"
```

Comment:
it's failing on ubuntu because you need to handle the different service naming depending on the current os, which is just `postgresql` in that case

you can add a conditional task based on `os_family` which equals to `Debian` when running on ubuntu/debian hosts